src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -88,6 +88,13 @@ func (r *ClusterReconciler) Reconcile(
 		return ctrl.Result{}, fmt.Errorf("unable to retrieve Cluster resource: %w", err)
 	}
 
+	managedAnnotationKey := redpandav1alpha1.GroupVersion.Group + "/managed"
+	if managed, exists := redpandaCluster.Annotations[managedAnnotationKey]; exists && managed == "false" {
```

Comment:
NIT
```suggestion
	if managed, exists := redpandaCluster.Annotations[managedAnnotationKey]; exists && !strconv.ParseBool(managed) {
```

Reply:
I was thinking of limiting the input to "false" specifically.